controllers/pipeline/pipeline_adapter.go

```diff
@@ -36,6 +36,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	requeueDelay = 10 * time.Second
```

Comment:
I reference the time in https://github.com/redhat-appstudio/managed-gitops/tree/main/appstudio-controller/controllers/appstudio.redhat.com when they meet some intermittent error, but I'd like to know your ideas. Thanks.

Reply:
Removed the manual delay since we can't implement it currently due to the issue in https://github.com/redhat-appstudio/operator-goodies/blob/main/reconciler/handler.go#L27.